src/main/java/frc/team997/lib/util/DashboardChooser.java

```diff
@@ -48,20 +48,29 @@
     private Logger<T> logger;
     private final boolean logChanges;
 
+    /**
+     * Interface implemented by all enums which DashboardChooser can automatically create a chooser
+     * from.
```

Comment:
Slightly awkward, maybe one of these?
Interface used by DashboardChooser to automatically create choosers from enum subclasses?
Interface implemented by enums which allows DashboardChooser to automatically create choosers?